ocs_ci/ocs/machine.py

```diff
@@ -138,6 +140,63 @@ def get_machinesets():
     return machine_sets
 
 
+def get_machine_from_machineset(machine_set):
+    """
+    Get the machine name from its associated machineset
+
+    Args:
+        machine_set (str): Name of the machine set
+
+    Returns:
+        List: Machine names
+    """
+    machine_objs = get_machine_objs()
+    machine_set_list = []
+    for machine in machine_objs:
+        if machine.get().get(
+                'metadata'
```

Comment:
please create a function to get machine name and use it here and other places

Reply:
This function is for the same purpose. It gets the machine name from the given machine set name. 
Please let me know if you are pointing to a different one.